packages/utils/src/process/resourceLogger.ts

```diff
@@ -3,9 +3,18 @@ import { memoryUsage, resourceUsage } from 'process'
 type LogType = (msg: string) => void | Promise<void>
 
 function createResourceLog(log: LogType) {
-  const used = memoryUsage()
-  const usage = resourceUsage()
-  log(`Process stats: mem ${used.rss / 1024}k (max: ${usage.maxRSS / 1024}k) ` + `cputime: ${usage.userCPUTime}`)
+  const resourcesUsed = resourceUsage()
+  // reported as KiloBytes
+  const maxUsedMemory = resourcesUsed.maxRSS / 1024
```

Comment:
It's up to you, but in these cases I suggest choosing variable names to reflect unit, e.g. `maxUsedMemoryMB`

Reply:
ok done